README.md

```diff
@@ -23,18 +23,15 @@ To run locally in a Docker container, execute:
 
     make run
 
+This command will run learnpython.org website by default
+
 To run a specific website, run with the DEFAULT_DOMAIN option set, as follows:
 
     DEFAULT_DOMAIN=learn-js.org make run
 
 By default, the server process will run at http://localhost:5000.
 
-Make build will build a Dockerfile and make run will default to run with `learnpython.org`.
-
-The web server will locally compile and load all Markdown files into memory. The development version of the web server
-requires that you specify the domain you are working on, e.g.:
```

Comment:
I removed this line as no example is given and it's explained in the previous lines